lib/extensions/lettering.py

```diff
@@ -174,23 +175,19 @@ def on_font_changed(self, event=None):
         self.settings.font = font.id
         self.scale_spinner.SetRange(int(font.min_scale * 100), int(font.max_scale * 100))
 
-
-        #Update the back_and_forth checkbox according to the chosen font
+        # Update the back_and_forth checkbox according to the chosen font
+        available_variants = font.available_variants()
         if font.reversible:
-            #by default
             self.back_and_forth_checkbox.Enable()
-            #TODO HERE :
-            #if (the reverse variant file does not exist for the variant):
-                #One svg variant file
-            #    self.back_and_forth_checkbox.SetValue(False)
-            #else:
-                #by default
-            #    self.back_and_forth_checkbox.SetValue(bool(self.settings.back_and_forth))
+            # TODO replace u"←" with something more clever
+            if u"←" in available_variants:
```

Comment:
Can we add a function like `Font.has_variant()`?  I think that might be cleaner.

Reply:
Yes, right :) Naming things again. I wasn't happy with that name but didn't know how to call it. Thanks :)